File: IUST 4th Sem/Searching in Array/linear search.cpp
#include <iostream>
using namespace std;

int main() {
    int arr[100];
    int n;
    cout << "Enter the Size of Array : ";
    cin >> n;
    
    cout << "Enter the Elements of Array : ";
    for(int i=0; i<n; i++){
        cin >> arr[i];
    }
    
    int find;
    cout << "Enter the Element you want to find : ";
    cin >> find;
    
    for(int i=0; i<n; i++){
        if(arr[i] == find){
            cout << "Element is in Array : " << arr[i];
        }
    }

    return 0;
}

File: IUST 4th Sem/Data Structure in Python/Searching in Array/linear search.py
def Linear_Search(arr, n, find):
    for i in range(0, n):
        if(arr[i] == find):
            return i;
    return -1;

arr = [1, 2, 3, 4, 5, 6, 7, 8, 9]
n = len(arr)
find = int(input("Enter the Element to Find : "))

result = Linear_Search(arr, n, find)
if(result == -1):
    print("Element is not in Array")
else:
    print("Element is in Array at Index : ", result)
